docs/reference-guides/slotfills/plugin-sidebar.md

```diff
@@ -1,30 +1,68 @@
 # PluginSidebar
 
-This slot allows for adding items into the Gutenberg Toolbar.
-Using this slot will add an icon to the bar that, when clicked, will open a sidebar with the content of the items wrapped in the `<PluginSidebar />` component.
+This slot allows adding items to the tool bar of either the Post or Site editor screens.
+Using this slot will add an icon to the toolbar that, when clicked, opens a panel with containing the items wrapped in the `<PluginSidebar />` component.
 
 ## Example
 
-```js
+```jsx
+import { __ } from '@wordpress/i18n';
+import { PluginSidebar } from '@wordpress/edit-post';
```

Comment:
Maybe this component should be imported from `@wordpress/editor` instead as per [Editor: Unified Extensibility APIs in 6.6](https://make.wordpress.org/core/2024/06/18/editor-unified-extensibility-apis-in-6-6/)

Reply:
oh, of course! Thanks for catching that.

Reply:
```suggestion
import { PluginSidebar } from '@wordpress/editor';
```